roles/openshift_hosted/tasks/router.yml

```diff
@@ -73,6 +73,20 @@
     resource_name: hostnetwork
   with_items: "{{ openshift_hosted_routers }}"
 
+- name: Set additional permissions for router service account if seamless reloads enabled.
+  oc_adm_policy_user:
+    user: "system:serviceaccount:{{ item.namespace }}:{{ item.serviceaccount }}"
+    namespace: "{{ item.namespace }}"
+    resource_kind: scc
+    resource_name: privileged
+    state: "{{ 'present' if openshift_hosted_router_seamless_reload else 'absent' }}"
+  with_items: "{{ openshift_hosted_routers }}"
+
+- name: Configure seamless router restart
+  set_fact:
+    openshift_hosted_router_edits: "{{ openshift_hosted_router_seamless_edits | union(openshift_hosted_router_edits) }}"
```

Comment:
Don't use set_fact here.  It's not a good practice to set_fact an existing variable name.

Reply:
You can apply the following to "Create OpenShift router" task:

```yaml
  edits: "{{ l_openshift_hosted_router_edits | union(item.edits) }}"
vars:
  l_ohr_seemless_edits: "{{ (openshift_hosted_router_seamless_reload | bool) | ternary(openshift_hosted_router_seamless_edits, []) }}"
  l_openshift_hosted_router_edits: "{{ openshift_hosted_router_edits | union(l_ohr_seemless_edits) }}"
```